service-api/src/main/java/greencity/dto/location/CoordinatesDto.java

```diff
@@ -14,11 +14,30 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
-@EqualsAndHashCode
 @ToString
 public class CoordinatesDto {
     @NotBlank
     private double latitude;
     @NotBlank
     private double longitude;
+
+    private static final double DELTA = 0.001;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        CoordinatesDto that = (CoordinatesDto) o;
+        return Math.abs(this.latitude - that.latitude) <= DELTA
+            && Math.abs(this.longitude - that.longitude) <= DELTA;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
The same here
it brakes hashCode() and hashCode() contract
`Equals Consistency: If two objects are equal according to the equals() method, then the hashCode() method must return the same value for them.`